lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-job.rb

```diff
@@ -0,0 +1,12 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_AnsibleTower_ConfigurationManager_Job < MiqAeServiceOrchestrationStack
+    expose :job_template, :association => true
+    expose :refresh_ems
+
+    def self.create_job(template, args = {})
+      template_object = template.instance_variable_get('@object')
```

Comment:
The other way to do this is to use the `id` of the template which will match the active_record object.  `template_object = ConfigurationScript.find_by(:id => template.id)`